gym/tf_od_api/workspace/train_core/pre-trained-models/efficientdet_d0/pipeline.config

```diff
@@ -0,0 +1,191 @@
+      conv_hyperparams {
+        regularizer {
+          l2_regularizer {
+            weight: 3.9999998989515007e-05
+          }
+        }
+        initializer {
+          truncated_normal_initializer {
+            mean: 0.0
+            stddev: 0.029999999329447746
+          }
+        }
+        activation: SWISH
+        batch_norm {
+          decay: 0.9900000095367432
+          scale: true
+          epsilon: 0.0010000000474974513
+        }
+        force_use_bias: true
+      }
+      bifpn {
+        min_level: 3
+        max_level: 7
+        num_iterations: 3
+        num_filters: 64
+      }
+    }
+    box_coder {
+      faster_rcnn_box_coder {
+        y_scale: 1.0
+        x_scale: 1.0
+        height_scale: 1.0
+        width_scale: 1.0
+      }
+    }
+    matcher {
+      argmax_matcher {
+        matched_threshold: 0.5
+        unmatched_threshold: 0.5
+        ignore_thresholds: false
+        negatives_lower_than_unmatched: true
+        force_match_for_each_row: true
+        use_matmul_gather: true
+      }
+    }
+    similarity_calculator {
+      iou_similarity {
+      }
+    }
+    box_predictor {
+      weight_shared_convolutional_box_predictor {
+        conv_hyperparams {
+          regularizer {
+            l2_regularizer {
+              weight: 3.9999998989515007e-05
+            }
+          }
+          initializer {
+            random_normal_initializer {
+              mean: 0.0
+              stddev: 0.009999999776482582
+            }
+          }
+          activation: SWISH
+          batch_norm {
+            decay: 0.9900000095367432
+            scale: true
+            epsilon: 0.0010000000474974513
+          }
+          force_use_bias: true
+        }
+        depth: 64
+        num_layers_before_predictor: 3
+        kernel_size: 3
+        class_prediction_bias_init: -4.599999904632568
+        use_depthwise: true
+      }
+    }
+    anchor_generator {
+      multiscale_anchor_generator {
+        min_level: 3
+        max_level: 7
+        anchor_scale: 4.0
+        aspect_ratios: 1.0
+        aspect_ratios: 2.0
+        aspect_ratios: 0.5
+        scales_per_octave: 3
+      }
+    }
+    post_processing {
+      batch_non_max_suppression {
+        score_threshold: 9.99999993922529e-09
+        iou_threshold: 0.5
+        max_detections_per_class: 100
+        max_total_detections: 100
+      }
+      score_converter: SIGMOID
+    }
+    normalize_loss_by_num_matches: true
+    loss {
+      localization_loss {
+        weighted_smooth_l1 {
+        }
+      }
+      classification_loss {
+        weighted_sigmoid_focal {
+          gamma: 1.5
+          alpha: 0.25
+        }
+      }
+      classification_weight: 1.0
+      localization_weight: 1.0
+    }
+    encode_background_as_zeros: true
+    normalize_loc_loss_by_codesize: true
+    inplace_batchnorm_update: true
+    freeze_batchnorm: false
+    add_background_class: false
+  }
+}
+train_config {
```

Comment:
we need to keep in mind we will have to add more data augmentation options in the future, but I guess we can leave it like that for now. 

Also, couldn't `random_horizontal_flip` and `random_scale_crop_and_pad_to_square` all be inside the same `data_augmentation_options` rather than in two separate ones?

Reply:
I can not answer this one